test/functional/fixtures/concurrency/test.js

```diff
@@ -114,6 +114,15 @@ if (config.useLocalBrowsers) {
             });
         }
 
+        if (!config.proxyless) {
+            it('Should run tests concurrently with Role', function () {
+                return run('chrome:headless --no-sandbox', 2, './testcafe-fixtures/role-test.js')
+                    .then(() => {
+                        expect(timeline.getData()).eql(['/fixtures/concurrency/pages/first-page.html', '/fixtures/concurrency/pages/second-page.html']);
```

Comment:
A long time ago, the `timeline` variable contains only time data . Now, it's not so.
Let's rename it to `testInfo`.